### PR TITLE
ci(aiohttp): move to riot

### LINKED ISSUE #1370: latest pytest is causing most of the async tests to be skipped
unfortunately this is not an error :(

I have a fix here: https://github.com/DataDog/dd-trace-py/pull/1369/files#diff-b91f3d5bd63fcd17221b267e851608e8L186

to be merged here: https://github.com/DataDog/dd-trace-py/pull/1319

### PR DESCRIPTION
**Pull request by P403n1x87**
## Description

Fixes #1370

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -971,6 +971,47 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={"urllib3": ["~=1.22.0", ">=1.23,<1.27", latest]},
             command="pytest {cmdargs} tests/contrib/urllib3",
         ),
+        Venv(
+            name="aiohttp",
+            command="pytest {cmdargs} tests/contrib/aiohttp",
+            pkgs={"pytest": [">=3"], "pytest-aiohttp": [latest]},
+            venvs=[
+                Venv(
+                    pys=["3.6"],
```
Comment (Yun-Kim) [inline riotfile.py:980]: should this include py 3.5 as well?

Comment (P403n1x87) [reply riotfile.py:980]: somehow all the tests were being skipped with 3.5, even with tox 🤷

**Review by Yun-Kim — DISMISSED** [review]

**Review by nizox — APPROVED** [review]

File: riotfile.py (hunk 1 above)
Comment (brettlangdon) [reply riotfile.py:977]: can we leave a comment here, otherwise someone might try to add 3.5 in the future

**Review by brettlangdon — APPROVED** [review]

**Comment by P403n1x87** [comment]
Closing for now as this requires enhancements in riot (i.e. install the dev package without `-e`).

**Pull request closed without merging**